applications/tari_app_grpc/src/conversions/new_block_template.rs

```diff
@@ -71,10 +71,10 @@ impl TryFrom<grpc::NewBlockTemplate> for NewBlockTemplate {
 
     fn try_from(block: grpc::NewBlockTemplate) -> Result<Self, Self::Error> {
         let header = block.header.clone().ok_or_else(|| "No header provided".to_string())?;
-        let total_kernel_offset =
-            BlindingFactor::from_bytes(&header.total_kernel_offset).map_err(|err| err.to_string())?;
-        let total_script_offset =
-            BlindingFactor::from_bytes(&header.total_script_offset).map_err(|err| err.to_string())?;
+        let total_kernel_offset = BlindingFactor::from_bytes(&header.total_kernel_offset)
+            .map_err(|err| format!("total_kernel_offset {}", err.to_string()))?;
+        let total_script_offset = BlindingFactor::from_bytes(&header.total_script_offset)
+            .map_err(|err| format!("total_script_offset {}", err.to_string()))?;
```

Comment:
nit: Same here, no need to allocate an additional string on the heap when using format!